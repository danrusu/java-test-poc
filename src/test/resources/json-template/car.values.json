{
  "model": "911",
  "engine.hp": 300,
  "engine.isDiesel": true,
  "engine.manufacturer.location": "Germany"
}