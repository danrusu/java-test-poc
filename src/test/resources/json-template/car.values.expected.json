{
  "model": "911",
  "engine": {
    "hp": 300,
    "isDiesel": true,
    "manufacturer": {
      "name": "Porsche",
      "location": "Germany"
    }
  }
}